src/interfaces.d.ts

```diff
@@ -1,7 +1,13 @@
 import * as React from 'react';
-import {Ref} from "react";
+import { Ref } from 'react';
 
-export interface ReactFocusLockProps<ChildrenType = React.ReactNode, LockProps = Record<string, any>> {
+// @FIXME: This was not defined earlier
+type FocusOptions = any;
```

Comment:
⚠️ Please note this.

Reply:
Why no tool consuming this file complained 😭

Reply:
Because no tool was consuming it. TypeScript files were published as-is. They still are, in fact. Only now they sit in a directory that happens to be linted.